eng/pipelines/mgmt.yml

```diff
@@ -0,0 +1,38 @@
+# External variables Settable at queue time:
+# "pomFile" : "pom.xml".   This is the project object model file for Maven.
```

Comment:
It doesn't look like the pomFile variable is used either.

Reply:
good catch